python/cuda_parallel/tests/test_scan_api.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+
+def test_scan_max():
+    # example-begin scan-max
+    import cupy as cp
+    import numpy as np
+
+    import cuda.parallel.experimental.algorithms as algorithms
```

Comment:
Why not put these imports at the top of the test file?

Reply:
Because the code within this test will be rendered into documentation examples (and we want those to include the imports used)

Reply:
See https://github.com/NVIDIA/cccl/blob/e5b92a229bdb06bf61c37ba262cb26e9d7fc6c89/python/cuda_parallel/cuda/parallel/experimental/algorithms/scan.py#L150